package.json

```diff
@@ -22,7 +22,7 @@
     "@next-auth/prisma-adapter": "^0.4.4-canary.81",
     "@next/font": "13.1.6",
     "@nextui-org/react": "^2.1.13",
-    "@prisma/client": "^5.10.2",
+    "@prisma/client": "^5.0.0",
```

Comment:
Is there a reason we downgraded? If they are current running 5.10 or higher in the prod site, this could cause conflicts.

Reply:
no, testing for some error.